septentrion/versions.py

```diff
@@ -17,3 +17,6 @@ def from_string(cls, version_string: str) -> "Version":
         except (AssertionError, ValueError) as exc:
             raise exceptions.InvalidVersion(str(exc)) from exc
         return cls(version_tuple=version_tuple, original_string=version_string)
+
+    def __str__(self):
+        return self.original_string
```

Comment:
An alternative would be to do `'.'.join(self.version_tuple)`, but I think keeping the original string is less confusing for users.